ui/hooks/useProposeFundPendingCampaign.ts

```diff
@@ -56,18 +61,32 @@ export const useFundPendingCampaign = (campaign: Campaign | null) => {
         },
       }
 
+      const msg = {
+        propose: {
+          title: `Activate DAO Up! campaign`,
+          description: `Send ${prettyPrintDecimal(amount)} ${
+            campaign.govToken.symbol
+          } to the [${campaign.name}](https://daoup.zone/campaign/${
+            campaign.address
+          }) campaign on DAO Up! in order to launch it.`,
+          msgs: [cosmMsg],
+        },
+      }
+
       try {
-        const msg = {
-          propose: {
-            title: `Activate DAO Up! campaign`,
-            description: `Send ${prettyPrintDecimal(amount)} ${
-              campaign.govToken.symbol
-            } to the [${campaign.name}](https://daoup.zone/campaign/${
-              campaign.address
-            }) campaign on DAO Up! in order to launch it.`,
-            msgs: [cosmMsg],
-          },
-        }
+        const daoProposalDeposit = Number(dao.config?.proposal_deposit)
+        if (!isNaN(daoProposalDeposit) && daoProposalDeposit > 0)
+          await client.execute(
+            walletAddress,
+            campaign.govToken.address,
+            {
+              increase_allowance: {
+                amount: dao.config.proposal_deposit,
+                spender: dao.address,
```

Comment:
Is this spender field being set correctly? Getting this error when testing:
<img width="519" alt="image" src="https://user-images.githubusercontent.com/30676292/154368797-7a352301-c2e2-42c5-b7ee-7911d208ac59.png">

Reply:
I am goon. One moment.